### ISSUE TITLE
Review text classification interface

### ISSUE THREAD

**Issue by ybbaigo** [ybbaigo/sklearn-plus#7]

**Comment by liuxiaoan8008** [comment]
1. Must add 'self.model_config = locals()' on the top of __init__ function.(It is used for model hyparameters save and load.)

2. Some tensors we care in training and predicting, please follow the rules below:
* use 'self.input_x' to represent for input x tensor.
* use 'self.input_y' to represent for input y tensor.
* use 'self.scores' to represent for raw model output.
* use 'self.logits' to represent for model output probability.
* use 'self.predictions' to represent for model predictions.
* use 'self.loss' to represent for model loss.

3. Add 'self.' in front of a tensor which you care and maybe used in training and predicting.

4. Some rules for regularization tricks:
* use 'self.dropout_keep_prob' to represent for dropout probability.
